recipes/fmilibrary/all/patches/2.4.1-010-install-runtime-to-bin.patch

```diff
@@ -0,0 +1,25 @@
+From 1c1fdb0d864682cb85433fc29cacbe829393eb1d Mon Sep 17 00:00:00 2001
+From: Joakim Haugen <joakim.haugen@sintef.no>
+Date: Mon, 30 Oct 2023 13:12:34 +0100
+Subject: [PATCH] Install runtime to bin
+
+---
+ CMakeLists.txt | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index cc2a25c..0bb24f6 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -304,7 +304,7 @@ file(COPY "${FMILIBRARYHOME}/Config.cmake/fmilib.h" DESTINATION "${FMILibrary_BI
+ install(TARGETS ${FMILIB_TARGETS}
+ 	ARCHIVE DESTINATION lib
+ 	LIBRARY DESTINATION lib
+-	RUNTIME DESTINATION lib
++	RUNTIME DESTINATION bin
```

Comment:
I would prefer moving DLL from lib to bin, using `shutil.move` than creating a patch. Patches are not portable from a version to another in case that file change, and we will become maintainers of this file once merged.

Reply:
I will ensure this is done in the recipe instead.